modules/virt-creating-linux-bridge-nad-cli.adoc

```diff
@@ -49,7 +50,8 @@ spec:
 <5> The name of the Linux bridge configured on the node.
 <6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
 <7> Optional: The VLAN tag. No additional VLAN configuration is required on the node network configuration policy.
-<8> Optional: Indicates whether the VM connects to the bridge through the default VLAN. The default value is `true`.
+<8> Optional: Specifies the state of the container interface (virtual ethernet device peer inside the network namespace). When set to `true`, the container interface is down. The default value is `false`. The `disableContainerInterface` field is relevant for IPv6 single-stack and IPv4/IPv6 dual-stack environments. You cannot use IP Address Management (IPAM) when this field is set to `false`.
```

Comment:
I suggest we remove this callout completely, and just leave the attribute unexplained.

The explanation of what the flag does is correct, but it is not relevant to the user - it only explains implementation details of KubeVirt's network binding.

This flag should be always set to "true". We do not support IPAM with bridge CNI on downstream, so the fact that setting it to "true" would break IPAM is unimportant. And while it is not required to have it on IPv4 single-stack, it does no harm setting it.

@ormergi please keep me honest here

Reply:
Correct :slightly_smiling_face:

Reply:
I have removed the callout description and just kept the field in the YAML manifest